test/DynamoCoreWpfTests/ViewExtensions/DocumentationBrowserViewExtensionTests.cs

```diff
@@ -103,6 +103,48 @@ public void ShowingStartPageHidesBrowser()
             Assert.AreEqual(Visibility.Hidden, visibilityAfterShowStartPageEvent);
         }
 
+        /// <summary>
+        /// This test validates that the Virtual Directory that will be created with WebView2 exists so the images will be loaded for a package node documentation
+        /// </summary>
+        [Test]
+        public void CanCreatePackageNodeDocumentationAndLoadImages()
+        {
+            // Arrange
+            RaiseLoadedEvent(this.View);
+
+            var testDirectory = GetTestDirectory(this.ExecutingDirectory);
+            var localImagePath = Path.Combine(testDirectory, @"core\docbrowser\pkgs\PackageWithNodeDocumentation\doc\icon.png");
+            var packageDocPath = Path.GetDirectoryName(localImagePath);
+
+            var docBrowserviewExtension = this.View.viewExtensionManager.ViewExtensions.OfType<DocumentationBrowserViewExtension>().FirstOrDefault();
+            var browserView = docBrowserviewExtension.BrowserView;
+
+            var nodeName = "Package.Hello";
+            var nodeRename = "New node name";
+            var expectedImageContent = String.Format(@"<img id='drag--img' class='resizable--img'  src=""http://appassets/{0}"" alt=""Dynamo Icon image"" />", Path.GetFileName(localImagePath));
+
+            // Act
+            this.ViewModel.ExecuteCommand(
+                 new DynamoModel.CreateNodeCommand(
+                     Guid.NewGuid().ToString(), nodeName, 0, 0, false, false)
+                 );
+
+            var node = this.ViewModel.Model.CurrentWorkspace.Nodes.FirstOrDefault();
+            node.Name = nodeRename; // Forces original name header to appear 
+            var nodeAnnotationEventArgs = new OpenNodeAnnotationEventArgs(node, this.ViewModel);
+
+            docBrowserviewExtension.HandleRequestOpenDocumentationLink(nodeAnnotationEventArgs);
+            var htmlContent = GetSidebarDocsBrowserContents();
+            htmlContent = htmlContent.Replace(@"%5C", "/");
```

Comment:
please add a comment

Reply:
Added comment, please check the next commit: https://github.com/DynamoDS/Dynamo/pull/13703/commits/f00db5b919450b0ccdafac4c9f90a8827b5d80c1